src/engine/configuration.cpp

```diff
@@ -134,6 +134,11 @@ namespace {
         // from previous runs) will be silently overwritten
         std::optional<std::string> scriptLog;
 
+        // If this value is set to `true`, the script log will contain scripts that have
```

Comment:
I'm confused. So if I set this to true, will the script log work as it used to before this PR again? Logging everything even stuff inside actions etc? From the description of the PR, it sounded like it would just add tags of whether it was synced or sent to remote or not, but it would still behave the same way as if it is false.

Reply:
To add to this, I think it could be good to have a setting that does turn it back the way it was with logging everything. In case an advanced user (or us) wants to copy a line of code inside an action and make something of their own with it.

Reply:
I made it a bit more obvious in the comment now.

`false`:  The ScriptLog is missing generated scripts as we define it wherever they occur (with Actions right now)
`true`:  The ScriptLog behaves the same as it did before the PR _and_ the `--[[sync]]`, `--[[generated]]` etc tags are added.

So there is no way to get the generated scripts without them being marked as generated